tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -0,0 +1,116 @@
+import platform
+
+from envparse import env
+
+
+plt = platform.platform()
+# TODO: in integration test for PR#411. Add 'oracle-logos' and 'centos-logos'
+#  to (ol7pkgs, ol8pkgs) and (cos7pkgs, cos8pkgs) respectively
+#  as these missing packages are a result of a different cause
+#  but still missing after a rollback
+ol7pkgs = ["oracle-release-el7", "usermode", "rhn-setup", "python-syspurpose"]
```

Comment:
I would the variable so it's in uppercase and words are separated by underscore.
```suggestion
OL_7_PKGS = ["oracle-release-el7", "usermode", "rhn-setup", "python-syspurpose"]
```

Reply:
Sure, good point